docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -268,12 +269,22 @@ Note that only one reporter can write to `stdout`. All other reporters must outp
 
 ### -s \<path\>, --screenshots \<path\>
 
-Enables screenshot capturing and specifies the directory where screenshots are saved.
+Enables screenshot capturing and specifies the root directory where screenshots are saved.
 
 ```sh
 testcafe all tests/sample-fixture.js -s screenshots
 ```
 
+#### Path Patterns
+
+The captured screenshots are organized into subdirectories within the root directory. The screenshots' relative path and name are defined using the default screenshot path patterns:
+
+* `${DATE}\test-${TEST_INDEX}\${USERAGENT}\{$SCREENSHOT_INDEX}.png` if the [quarantine mode](../../using-testcafe/command-line-interface.md#-q---quarantine-mode) is disabled;
+* `${DATE}\test-${TEST_INDEX}\run-${QUARANTINE_ATTEMPT}\${USERAGENT}\{$SCREENSHOT_INDEX}.png` if the [quarantine mode](../../using-testcafe/command-line-interface.md#-q---quarantine-mode) is enabled.
+* `${DATE}\test-${TEST_INDEX}\${USERAGENT}\errors\{$SCREENSHOTINDEX}.png` if the [--screenshots-on-fails](#-s---screenshots-on-fails) option is specified.
```

Comment:
Add one more pattern for quarantine mode plus take screenshot on fails
`${DATE}\test-${TEST_INDEX}\run-${QUARANTINE_ATTEMPT}\${USERAGENT}\errors\{$SCREENSHOT_INDEX}.png`